### DISCUSSION TITLE
if else 问题

### DISCUSSION THREAD

**Discussion by ahank47** [ofajs/discussion#10] [Q&A]
`<x-if :value = "false">` 还是 `<x-if :value = false>` ?

**Comment by ahank47** [comment]
发现个问题， 这两个都不会报错，但是 false 后， `<x-if>` 后的 `<x-else>` 一起不生效

**Comment by ahank47** [comment]
玄学，单独提取出来是可以的，但是放在 thymeleaf 中 会有 if false 后 的 else 无法执行

**Reply by kirakiray** [reply]
可能和thymeleaf的模板语法有冲突
